tests/unit/routers/test_entity.py

```diff
@@ -655,3 +655,76 @@ def test_search_entities_multiple_entities_returned_no_query_params_geojson(
     assert "type" in result, "expecting geojson structure to have type attribute"
     assert result["type"] == "FeatureCollection"
     assert "features" in result, "expected features attribute"
+
+
+def test_search_entities_exclude_field(mocker, multiple_entity_models):
```

Comment:
This isn't a bad test as it checks whether the logic that happens after the get_entity_search function is applied. however the code you've added is in the get_entity_search function so it's not testing what you've added.